projects/aca-shared/src/lib/components/open-in-app/open-in-app.component.ts

```diff
@@ -35,7 +35,7 @@ export interface OpenInAppDialogOptions {
   encapsulation: ViewEncapsulation.None
 })
 export class OpenInAppComponent {
-  private redirectUrl: string;
+  private readonly redirectUrl: string;
   public window: Window & typeof globalThis = window;
```

Comment:
Any reason for this? Window is global so we can access everywhere to it in ts code. Also it does not shorten code because it is still just 'window' word.

Reply:
it was some sort of warkaround for something, no idea, but guess we can address later after figuring out why it's needed, I just added readonly as per IntelliJ suggestions